### PR TITLE
Slight standby implementation changes

### PR DESCRIPTION
**Pull request by eddievhfan1984**
-Adjusted standby debug line position (otherwise requires optics on)
-Added reset of SbyPressed state

Signed-off-by: Kyle Tekaucic <tekaucic.2@buckeyemail.osu.edu>

### CODE REVIEW

File: Orbitersdk/samples/ProjectApollo/src_sys/yaAGC/agc_engine.c [hunk 1]
```diff
@@ -1749,6 +1749,8 @@ agc_engine (agc_t * State)
 		  // Same with Standby
 		  if (0 == (State->InputChannel[032] & 020000))
 			  State->SbyPressed = 1;
+		  else
+			  State->SbyPressed = 0;
```
Comment (thewonderidiot) [inline Orbitersdk/samples/ProjectApollo/src_sys/yaAGC/agc_engine.c]: I recommend moving this check outside of the `while (ScalerCounter >= SCALER_OVERFLOW)` loop. Placing it here makes the check for PRO release only occasionally happen based on the current scaler time; in reality there's no clocks gating the reset so it happens immediately upon release. So you might get some slightly weird timing effects. Putting the check to do this just before the while loop should be fine.

Comment (eddievhfan1984) [inline Orbitersdk/samples/ProjectApollo/src_sys/yaAGC/agc_engine.c]: Done in the new commit.

**Comment by indy91** [comment]
Does the change look good to you @thewonderidiot ? I wouldn't know...

**Comment by thewonderidiot** [comment]
Ah, sorry, what I meant is that the setting and unsetting can't be in the same spot. The original code was in the right place, as PRO being depressed is only checked for by the hardware once every 1.28 seconds. It's just the release that is continuously monitored.

The code should be something like this:

```
if (1 == (State->InputChannel[032] & 020000))
   State->SbyPressed = 0;

...

while (ScalerCounter >= SCALER_OVERFLOW)
     {

...

        if (0 == (State->InputChannel[032] & 020000))
 		State->SbyPressed = 1;

...
```

**Comment by indy91** [comment]
Looks good to me now.

**Pull request merged**